### ISSUE TITLE
Seperate the single Autoplay option for Videos AND GIFs into two

### ISSUE THREAD

**Issue by RetroSunstar** [bluesky-social/social-app#5277]
Looking at [this post](https://bsky.app/profile/caliburn-ex.bsky.social/post/3l3vqk3wg6c26), having the option to disable autoplay for videos OR gifs instead of both could be a good idea

There are times a user may not want to have videos Autoplay, which they can disable in [settings](https://bsky.app/settings/accessibility) but this causes GIFs to stop autoplaying as well, which seems fairly inconvenient to hit play on every gif just because you don't want videos to autoplay

It's not bad to give the option to users who want it, essepcially with certain types of video content that the user may not want to view which doesn't necessarily extend to gifs 🩵🩷🩵🩷

(If it's not too much to ask???)

**Comment by pietrofxq** [comment]
I agree - videos also consume way more bandwidth than GIFs. I don't mind downloading some gifs but streaming every video passing by the timeline, especially in mobile networks, can be too much

**Comment by ThatOneUnoriginal** [comment]
I would like this since it seems like auto playing videos seems to trip up last.fm when scrolling through the timeline (at least on the Android version.)
